src/ucp/core/ucp_worker.c

```diff
@@ -887,6 +887,16 @@ static int ucp_worker_iface_support_keepalive(ucp_worker_iface_t *wiface)
                               UCT_IFACE_FLAG_EP_KEEPALIVE);
 }
 
+/* Compare target iface atomic capabilities to candidate's, and decide if candidate is better */
+static int ucp_worker_iface_is_better_atomic_caps(uct_iface_attr_t *cand_attr,
```

Comment:
Don't abbreviate (`cand`).
Possible alternatives: target - compare, attr - test_attr.

Reply:
not sure I understand, you mean switch the names?
can you write the full function signature?

Reply:
1. Abbreviations are bad (`cand`).
2. Let's replace the names. Instead of `cand_attr` - `target_attr`, can have some combination of `test_attr` / `compare_attr` / `target_attr`.

Reply:
notice that the second argument is already called `target_attr` (cause I had `candidate` which was compared to the `target`).
so if we rename the first argument to `target`, we should also rename the second argument.
maybe we should just call them `first` and `second`?

Reply:
`first` and `second` doesn't show the difference in their meaning.
How about `compare_attr` - `test_attr`?
To resemble something similar to https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_worker.c#L926